services/account/apps/api/src/controllers/v1/accounts-v1.controller.ts

```diff
@@ -40,10 +40,31 @@ export class AccountsControllerV1 {
    * @returns A promise that resolves to an Account object => {msaId, handle}.
    * @throws An error if the account cannot be found.
    */
-  async getAccount(@Param('msaId') msaId: string): Promise<AccountResponse> {
+  async getAccountForMsa(@Param('msaId') msaId: string): Promise<AccountResponse> {
     try {
       this.logger.debug(`Received request to get account with msaId: ${msaId}`);
-      return await this.accountsService.getAccount(msaId);
+      return await this.accountsService.getAccountForMsaId(msaId);
+    } catch (error) {
+      this.logger.error(error);
+      throw new HttpException('Failed to find the account', HttpStatus.BAD_REQUEST);
+    }
+  }
+
+  @Get('account/:publicKey')
```

Comment:
Had to add a prefix so that it wouldn't get confused with the GET :msaId  endpoint. Open to changing the prefix if you guys have a better name.